infra/charts/feast-feature-server/Chart.yaml

```diff
@@ -3,6 +3,7 @@ name: feast-feature-server
 description: Feast Feature Server in Go or Python
 type: application
 version: 0.45.0
+deprecated: true
```

Comment:
?

Reply:
should we be changing the actual helm chart? 
```suggestion
```

Reply:
@lokeshrangineni suggested and I tend to agree, we are deprecating it, no ? why not mark it in the chart as well, aren't we going to remove it in the future in a later version ?

Reply:
I am a bit confused,
There are 2 charts that are the same in different paths with confusing names:
1. infra/charts/feast-feature-server/Chart.yaml
2. infra/feast-helm-operator/helm-charts/feast-feature-server/Chart.yaml
Both exactly the same and both with the `name: feast-feature-server` its confusing it sounds like the helm operator is the feature server helm ? @tchughesiv @franciscojavierarceo

Reply:
that's the helm chart... we're only deprecating the helm-based operator

Reply:
we're ONLY deprecating this -
https://github.com/feast-dev/feast/tree/121617053344117cdbfbb480882b10cc176245ac/infra/feast-helm-operator

Reply:
@tmihalac i see where the confusion comes in...  if you look closely, its a sym-linked dir. this was done to simplify the operator builds and chart references. they are one in the same chart.
https://github.com/feast-dev/feast/tree/121617053344117cdbfbb480882b10cc176245ac/infra/feast-helm-operator/helm-charts